molo/profiles/templates/profiles/register.html

```diff
@@ -2,50 +2,58 @@
 {% load i18n static core_tags wagtailsettings_tags %}
```

Comment:
Please apply the things that I've been asking you to do in `change_password` in this file as well.